jpo-ode-plugins/src/main/java/us/dot/its/jpo/ode/plugin/j2735/timstorage/Position.java

```diff
@@ -15,21 +15,21 @@
  ******************************************************************************/
 package us.dot.its.jpo.ode.plugin.j2735.timstorage;
 
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.annotation.JsonPropertyOrder;
 
 import us.dot.its.jpo.ode.plugin.asn1.Asn1Object;
 
+@JsonIgnoreProperties(ignoreUnknown = true)
 @JsonPropertyOrder({ "lat", "llong", "elevation" })
 public class Position extends Asn1Object {
    private static final long serialVersionUID = 1L;
 
-   @JsonProperty("latitude")
+   @JsonProperty("lat")
    private String lat;
-
-   @JsonProperty("longitude")
+   @JsonProperty("long")
    private String llong; // TODO needs to be "long"
```

Comment:
Should this be long or llong? If long, maybe remove the TODO.

Reply:
The JSON declaration should be "long", as that's what is expected when decoding from JSON string or encoding to that string. I believe the TODO was in there referencing the fact that the property can't be called "long" in the code, but that's been in there for quite some time. Since the property is JSON encoding/decoding as "long" now, I'll remove the comment.